src/screens/DiscoveryScreenV2/fragment/UsersFragment.js

```diff
@@ -273,18 +288,7 @@ const UsersFragment = ({
       </View>
     );
 
-  return (
-    <View>
-      {!withoutRecent && (
-        <RecentSearch
-          shown={showRecentSearch || isFirstTimeOpen}
-          setSearchText={setSearchText}
-          setIsFirstTimeOpen={setIsFirstTimeOpen}
-        />
-      )}
-      {renderUsersItem()}
-    </View>
-  );
+  return <View>{renderUsersItem()}</View>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `RecentSearch` component is moved to the `renderUsersItem` function. This change seems to improve the code structure by encapsulating the rendering logic within the `renderUsersItem` function.
```diff
- return (
-   <View>
-     {!withoutRecent && (
-       <RecentSearch
-         shown={showRecentSearch || isFirstTimeOpen}
-         setSearchText={setSearchText}
-         setIsFirstTimeOpen={setIsFirstTimeOpen}
-       />
-     )}
-     {renderUsersItem()}
-   </View>
- );
+ return <View>{renderUsersItem()}</View>;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->